packages/wxt/src/core/clean.ts

```diff
@@ -1,44 +1,48 @@
 import path from 'node:path';
 import glob from 'fast-glob';
 import fs from 'fs-extra';
-import { consola } from 'consola';
 import pc from 'picocolors';
+import { InlineConfig } from '~/types';
+import { registerWxt, wxt } from './wxt';
 
 /**
  * Remove generated/temp files from the directory.
  *
- * @param root The directory to look for generated/temp files in. Defaults to `process.cwd()`. Can be relative to `process.cwd()` or absolute.
+ * @param config Optional config that will override your `<root>/wxt.config.ts`.
  *
  * @example
  * await clean();
  */
-export async function clean(root = process.cwd()) {
-  consola.info('Cleaning Project');
+export async function clean(config?: InlineConfig) {
+  await registerWxt('build', config);
+  wxt.logger.info('Cleaning Project');
+
+  const root = wxt.config.root;
 
   const tempDirs = [
     'node_modules/.vite',
     'node_modules/.cache',
     '**/.wxt',
-    '.output/*',
+    `${path.relative(root, wxt.config.outBaseDir)}/*`,
   ];
-  consola.debug('Looking for:', tempDirs.map(pc.cyan).join(', '));
+  wxt.logger.debug('Looking for:', tempDirs.map(pc.cyan).join(', '));
```

Comment:
Woops, there's a couple places outside this file that need to use `wxt.logger` as well. I'll do that in a separate PR. Good find